### ISSUE TITLE
CKV_GHA_7 - GitHub Actions workflow_dispatch inputs MUST be empty - why?

### ISSUE THREAD

**Issue by ruzickap** [bridgecrewio/checkov#3839]
**Describe the issue**

Hello guys.
checkov is reporting `CKV_GHA_7` in many GH Actions which are using `inputs:` and `workflow_dispatch:` like:

```yaml
name: insider-windows

on:
  workflow_dispatch:
    inputs:
      release_version:
        type: string
        description: Forced release version
      new_release:
        type: boolean
        description: Force new Release
...
```

https://github.com/VSCodium/vscodium/blob/51e8c9644717f552ed6f8767e028577418f5d8d9/.github/workflows/insider-windows.yml#L4-L5  

Is there a reason why `CKV_GHA_7` was created?

In official GitHub documentation using `inputs:` and `workflow_dispatch:` are supported:

* https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/
* https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs

Am I missing something ?

**Version**
 - Checkov Version:  2.2.30
